config/dev.exs

```diff
@@ -14,6 +14,11 @@ config :jumubase, JumubaseWeb.Endpoint,
   watchers: [node: ["node_modules/brunch/bin/brunch", "watch", "--stdin",
                     cd: Path.expand("../assets", __DIR__)]]
 
+# Configure Phauxth authentication
+config :phauxth,
+  token_salt: System.get_env("PHAUXTH_TOKEN_SALT") || "1J9hSjU4",
```

Comment:
Let's default to something like `String.duplicate("x", 8)` to make it perfectly clear that no real secrets are getting hard-coded here 😅

Reply:
Good idea. Although` "xxxxxxxx"` is shorter and equally clear. 😉

Reply:
Well, in some cases the token length matters (and might be a lot longer) and with that style you immediately see the length. But for 8 chars I agree.